programmers2/제일-작은-수-제거하기2.js

```diff
@@ -0,0 +1,5 @@
+function solution(arr) {
+    arr.splice(arr.indexOf(Math.min.apply(null,arr)), 1);
+    if( arr.length === 0 ) return [-1];
```

Comment:
공백 유의!
arr.length === 0 은 !arr.length로 바꿀 수 있습니다
```suggestion
    if (!arr.length) return [-1];
```